test/runtests.jl

```diff
@@ -17,4 +19,7 @@ using StatsFuns
         include("likelihoods/exponential.jl")
         include("likelihoods/negativebinomial.jl")
     end
+    @testset "expectations" begin
+        include("expectations.jl")
+    end
```

Comment:
```suggestion
    include("expectations.jl")
```
not in separate folder, so fine to put by itself